Core/src/IfcGeometryConverter/GeometrySettings.h

```diff
@@ -163,46 +172,41 @@ namespace OpenInfraPlatform
 						dAngle += M_TWOPI;
 					}
 				}
-
-				bool areEqual(const double first,const double second) const
-				{
-					if (abs(first - second) < getPrecision()) 
-					{
-						return true;
-					}
-					return false;
-				}
-
-				bool areEqual(const carve::geom::vector<2>& first, const  carve::geom::vector<2>& second) const
+				/*!Verify two points of series whith precision
+				*\param[in]    first      The first point of serie
+				*\param[in]    second     The second point of serie
+				*\param[in]    precision  Defaul parameter equal getPrecision()
+				
+				*\return true if different is smaler then the precision [bool]
+				*\note function areEqual has 2 overloads
+				*/
+				bool areEqual(const double first,const double second,const double precision = getPrecision()) const
 				{
-					if (abs(first.x - second.x) < getPrecision() && abs(first.y - second.y) < getPrecision())
+					
+					if (abs(first - second) < precision) 
 					{
 						return true;
 					}
 					return false;

```

Comment:
```suggestion
					return abs(first - second) < precision;
```

Simpler syntax - a comparison returns a bool, so one can return it directly.

Reply:
Resolved: b2f7040